tests/migrations/test_writer.py

```diff
@@ -321,6 +322,29 @@ class IntEnum(enum.IntEnum):
             "default=migrations.test_writer.IntEnum(1))"
         )
 
+    def test_serialize_uuid(self):
+        uuid_a = uuid.UUID('5c859437-d061-4847-b3f7-e6b78852f8c8')
+        uuid_b = uuid.UUID('c7853ec1-2ea3-4359-b02d-b54e8f1bcee2')
+
+        self.assertSerializedResultEqual(
+            uuid_a,
+            ("uuid.UUID('" + str(uuid_a) + "')", {'import uuid'})
```

Comment:
Why not just put the actual string here, as in the model serialization below? E.g.:

	         self.assertSerializedResultEqual(
	             uuid_a,
	-            ("uuid.UUID('" + str(uuid_a) + "')", {'import uuid'})
	+            ("uuid.UUID('5c859437-d061-4847-b3f7-e6b78852f8c8')", {'import uuid'})
	         )
	         self.assertSerializedResultEqual(
	             uuid_b,
	-            ("uuid.UUID('" + str(uuid_b) + "')", {'import uuid'})
	+            ("uuid.UUID('c7853ec1-2ea3-4359-b02d-b54e8f1bcee2')", {'import uuid'})
	         )

This also keeps the test consistent with other tests.